core/templates/dev/head/editor/ExplorationSettings.js

```diff
@@ -247,16 +247,66 @@ oppia.controller('ExplorationSettings', [
     });
   };
 
-  $scope.publicizeExploration = function() {
-    explorationRightsService.saveChangeToBackend({is_publicized: true});
+  var openModalForModeratorAction = function(action) {
+    warningsData.clear();
+
+    $http.get('/moderatorhandler/email_draft/' + action).then(function(response) {
+      // An empty draft email body indicates that no email will be sent.
```

Comment:
This was unclear; the comment meant that if the email draft was empty, the email form wouldn't be shown in the UI at all. I clarified it.